tests/test_envs.py

```diff
@@ -52,127 +49,148 @@ def test_run(env=None, num_steps=100, verbose=False, **kwargs):
+        for env_name in ENVS:
+            env = ngym.make(env_name)
+            _test_trialenv(env)
 
 
-def test_seeding(env=None, seed=0):
+def _test_seeding(env):
     """Test if environments are replicable."""
-    if env is None:
-        env = ngym.all_envs()[0]
-
-    if isinstance(env, str):
-        kwargs = {"dt": 20}
-        env = make_env(env, **kwargs)
-    elif not isinstance(env, gym.Env):
-        msg = f"{type(env)=} must be a string or a gym.Env"
-        raise TypeError(msg)
-
-    env.seed(seed)
-    env.reset()
-    ob_mat = []
-    rew_mat = []
-    act_mat = []
-    for _ in range(100):
-        action = env.action_space.sample()
-        ob, rew, terminated, _truncated, _info = env.step(action)
-        ob_mat.append(ob)
-        rew_mat.append(rew)
-        act_mat.append(action)
-        if terminated:
-            env.reset()
-    ob_mat = np.array(ob_mat)
-    rew_mat = np.array(rew_mat)
-    act_mat = np.array(act_mat)
-    return ob_mat, rew_mat, act_mat
+    with warnings.catch_warnings():
+        warnings.filterwarnings("ignore", message=".*get variables from other wrappers is deprecated*")
+        warnings.filterwarnings("ignore", message=".*The environment creator metadata doesn't include `render_modes`*")
+        if env is None:
+            env = ngym.all_envs()[0]
+
+        if isinstance(env, str):
+            kwargs = {"dt": 20}
+            env = ngym.make(env, **kwargs)
+        elif not isinstance(env, gym.Env):
+            msg = f"{type(env)=} must be a string or a gym.Env"
+            raise TypeError(msg)
+
+        env.seed(SEED)
+        env.reset()
+        ob_mat = []
+        rew_mat = []
+        act_mat = []
+        for _ in range(100):
+            action = env.action_space.sample()
+            ob, rew, terminated, _truncated, _info = env.step(action)
+            ob_mat.append(ob)
+            rew_mat.append(rew)
+            act_mat.append(action)
+            if terminated:
+                env.reset()
+        ob_mat = np.array(ob_mat)
+        rew_mat = np.array(rew_mat)
+        act_mat = np.array(act_mat)
+        # FIXME: given the returns below, it seems as though this should be the helper function for the test below
+        # rather than its own test, except that the first env is chosen seemingly arbitrarily. This can be done in next
+        # fucntion instead to avoid the returns in an actual test. This should maybe be implemented for other tests here
+        # as well
+        return ob_mat, rew_mat, act_mat
 
 
 # TODO: there is one env for which it sometimes raises an error
 def test_seeding_all():
     """Test if all environments are replicable."""
     for env_name in sorted(ENVS_NOPSYCHOPY):
         print(f"Running env: {env_name}")
-        obs1, rews1, acts1 = test_seeding(env_name, seed=0)
-        obs2, rews2, acts2 = test_seeding(env_name, seed=0)
+        obs1, rews1, acts1 = _test_seeding(env_name)
```

Comment:
I'd catch the warning here, in the upper function (`test_seeding_all`) instead of in `_test_seeding`, as it's done already for `test_run_all` and `test_trialenv_all`